### PR TITLE
Play view

### PR DESCRIPTION
**Pull request by enihsgnir**
implementing play view

### CODE REVIEW

File: lib/views/play/widgets/board_square.dart [hunk 1]
```diff
@@ -29,13 +29,14 @@ class BoardSquare extends ConsumerWidget {
 
         if (cell.hasMine) {
           ref.read(cellNotifierProvider(pos).notifier).setOpen();
-          ref.read(logNotifierProvider.notifier).add(pos);
         } else {
           ref.read(cellNotifierProvider(pos).notifier).open();
         }
+        ref.read(logNotifierProvider.notifier).add(pos);
```
Comment (changhoji) [inline lib/views/play/widgets/board_square.dart:35]: I think can remove duplicate `ref.read(logNotifierProvider.notifier)`

**Review by changhoji — APPROVED** [review]
great test code!

File: lib/views/play/widgets/board_square.dart (hunk 1 above)
Comment (enihsgnir) [reply lib/views/play/widgets/board_square.dart:35]: which duplicates do you mean?

Comment (changhoji) [reply lib/views/play/widgets/board_square.dart:35]: I menat line 35 and line 39

Comment (enihsgnir) [reply lib/views/play/widgets/board_square.dart:35]: got it. i'll consider this in the next refactoring.

**Pull request merged**